@here/harp-mapview-decoder/lib/ThemedTileDecoder.ts

```diff
@@ -35,9 +38,10 @@ export abstract class ThemedTileDecoder implements ITileDecoder {
         data: ArrayBufferLike,
         tileKey: TileKey,
         projection: Projection
-    ): Promise<DecodedTile> {
+    ): Promise<DecodedTile | undefined> {
         if (this.m_styleSetEvaluator === undefined) {
-            return Promise.reject(new Error("No style is defined"));
+            logger.info("cannot decode tile, as there is not style available");
```

Comment:
would logger.warn or logger.error be better?

Reply:
I think its actually not an error, e.g. if we set a Theme to a Datasource that has not Styles for the Type of Data, we will run into this, which is not wrong.
Or if no theme has yet been supplied

Reply:
i'm not sure about using logger.info. I'd say it's a `warning` if we want the user to change its code because otherwise they are creating useless requests that are slow and don't produce anything.

Actually, I'm not sure what logger.info() is useful for, do we have any guidelines for our logger?